src/EnergyPlus/WeatherManager.hh

```diff
@@ -619,6 +621,7 @@ namespace WeatherManager {
 		// Member Constructor
```

Comment:
@rglazer2 If this member constructor isn't ever used (I doubt it is, but maybe...), then you can just delete the whole constructor instead of adding to it.